### PR TITLE
 Add .rpm package generation

### PR DESCRIPTION
**Pull request by pratid**
#### Reviewers

@jframos 
#### Description

Generation of a .rpm package
#### Testing

N/A

### CODE REVIEW

File: dashboard/lib/config.js [hunk 1]
```diff
@@ -14,83 +14,84 @@
...
-    };
-
+var config = {
+    config_file: path.join(__dirname, '/../config', name + '.yml'),
+    log_level: 'DEBUG',
+    listen_port: 3000,
+    secret:'ssshhh',
+    paths: {
+        reports_url: '/check/report',
+        reports_files: '/var/www/html/RegionSanityCheck'
```
Comment (jframos) [inline dashboard/lib/config.js:43]: Currently, we are moving generated reports to `/var/www/html/RegionSanityCheckv2`. If we configured this new path, we must change Jenkins configuration.

File: dashboard/tools/build/files/redhat/SPECS/fiware-fihealth-dashboard.spec [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+Version: %{_version}
+Release: %{_release}
+License: Apache
+Group: Applications/Engineering
+Vendor: Telefónica I+D
+BuildArch: noarch
+# Requires: nodejs (see %pre)
+
+%description
+Blah blah
```
Comment (jframos) [inline dashboard/tools/build/files/redhat/SPECS/fiware-fihealth-dashboard.spec]: Blah, blah? :flushed:

File: dashboard/lib/config.js (hunk 1 above)
Comment (pratid) [inline dashboard/lib/config.js:43]: These are default values. Any post-install customization should be done in dashboard.yml config fle (the user is warned about this after installing the package).

**Comment by jframos** [comment]
LGTM

**Comment by jframos** [comment]
What about the coverage?

**Comment by pratid** [comment]
No new functionality was added, apart from moving some "hardcoded" constants to the configuration file. Decrease of coverage appeared after merging the latest commits.

**Pull request merged**